package.json

```diff
@@ -54,15 +65,16 @@
         "@typescript-eslint/parser": "^5.44.0",
         "eslint": "^8.28.0",
         "langium-cli": "0.6.0-next.62c127c",
```

Comment:
Please update langium and langium-cli to 1.0.0

Reply:
Done